exercises/farewell-workflow/solution/Workflow/GreetingWorkflow.cs

```diff
@@ -0,0 +1,25 @@
+namespace Temporalio.Farewell.Workflow;
```

Comment:
#5 used `MyNamespace` but this one uses a more proper namespace. Is the difference intentional? Would like to see #5 get updated with a proper namespace too.